schematic/models/metadata.py

```diff
@@ -2,7 +2,7 @@
 import networkx as nx
 import json
 import re
-
+import synapseclient
```

Comment:
looks like we no longer need synapseclient import?

Reply:
Yup, good catch, you're right. Removed `synapseclient` and `re` imports, both of which weren't being used.